### DISCUSSION TITLE
Hardware accel help please

### DISCUSSION THREAD

**Discussion by crclark62** [blakeblackshear/frigate#5999] [Ask A Question] [answered]
I have tried to figure out what if any hardware acceleration I should use. I'm running Frigate on my server in Docker.
My hardware is quite odd for this but its what I have.
The server has a not so great integrated graphics card, it's a Matrox G200eR2 the server has dual Xeon CPU's and 64GB of ram.

Below is the cpu and ram info.
Thanks in advance.

![tower-info](https://user-images.githubusercontent.com/92645252/231236553-98bd79d6-0263-438a-895b-24299fded0a9.png)

**Comment by NickM-27** [comment, marked as answer]
you will need a discrete gpu to be able to use hwaccel

**Reply by crclark62** [reply]
Perfect, Thank you 
I just upgraded to 12 and its really great and Frigate runs fine the way it is.
